pkg/analyze/front/print_test.go

```diff
@@ -0,0 +1,64 @@
+package front
+
+import (
+	"bytes"
+	"github.com/scylladb/scylla-operator/pkg/analyze/symptoms"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"os"
+	"testing"
+)
+
+func TestPrint(t *testing.T) {
+	stdout := os.Stdout
+	r, w, _ := os.Pipe()
+	os.Stdout = w
+
+	defer func() {
+		os.Stdout = stdout
+	}()
```

Comment:
This will prevent you from running the tests in parallel (which should be the default here). Front should take a writer instead (it's good to have either way, see Maciej's comment https://github.com/zpp-2024-scylla-operator-analyze/scylla-operator/pull/28#discussion_r2056166363)